CMakeLists.txt

```diff
@@ -2366,6 +2366,9 @@ else()
 endif()
 
 # TagLib
+# TODO: Check for minimum required version?
+# See also: static_assert in src/track/taglib/trackmetadata_common.h
+# that verifies TAGLIB_MAJOR_VERSION/TAGLIB_MINOR_VERSION.
```

Comment:
I would expect a failure at configure time rather than build time for an outdated dependency, so I think it would be better to check the minimum version with CMake than static_assert.

Reply:
We can replace the static_assert with a configure check if someone provides it. Until then the static_assert is even more safe although less convenient.

Reply:
How is it more safe?

Reply:
Nearer at the code. Let's agree on just safe ;) We also have other version checks in the code.

But it would be preferable to check the minimum required version beforehand.

Reply:
> if someone provides it

Done: https://github.com/uklotzde/mixxx/pull/17

Reply:
In my experience, it is an unpleasant surprise to have a build fail because of an outdated dependency. I prefer to have the build system tell me up front that it will not work and why.

Reply:
Also I think it is easier for us to keep track of what our minimum required dependencies are in the build system than having to dig in the C++ code for that information.